src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &--0 &__star:nth-child(-n+4) {
```

Comment:
apply styles for all amount of cases (just reuse your solution from a previous task)

Reply:
https://github.com/mate-academy/layout_catalog/pull/3689/files#r1341318983
fix it please